lib/PhasarLLVM/Pointer/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-file(GLOB_RECURSE POINTER_SRC *.h *.cpp)
+file(GLOB POINTER_SRC *.cpp TypeGraphs/*.cpp)
```

Comment:
folder, "TypeGraphs" is not present in the repository

Reply:
True, we recently removed it, thanks!